core/classes/Adapter.js

```diff
@@ -118,7 +121,32 @@ module.exports = class Adapter {
     return trx ? query.transacting(trx) : query;
   }
 
+  _newTable (schemaKey, schema) {
+    return this.knex.schema.createTable(schemaKey, (table) => {
+      table.increments("id").primary();
+      _.forEach(schema.properties, (value, key) => {
+        if (key === "id") return;
+        if (!(POSTGRES_TYPES.includes(value.type))) {
+          throw new Error(`Incorrect data type ${value.type} of field ${key} in ${schemaKey}`);
+        }
+        const attribute = table[value.type](key);
+        if ((Array.isArray(schema.required)) && schema.required.includes(key)) {
```

Comment:
Parenthesis around `(Array.isArray(schema.required))` are not required and should be deleted